src/main/java/net/glowstone/generator/objects/trees/RedwoodTree.java

```diff
@@ -45,6 +47,7 @@ protected final void setLeavesHeight(int leavesHeight) {
 
     @Override
     public boolean canPlace(int baseX, int baseY, int baseZ, World world) {
+        Set<Material> overridableSet = new HashSet<>(overridables);
```

Comment:
I think we should convert the base class' overridables to a HashSet, to prevent allocations every call.

Reply:
You can redesign the class to achieve better optimization. Currently, such allocation also improves the efficiency of the single function indeed.